src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,139 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String VALID_PASSWORD = "1234567";
+    private static final String VALID_PASSWORD_MIN_LENGTH = "123456";
+    private static final String INVALID_PASSWORD = "12345";
+    private static final String EMPTY_PASSWORD = "";
+    private static final String VALID_LOGIN = "userLogin";
+    private static final String VALID_LOGIN_MIN_LENGTH = "userLo";
+    private static final String INVALID_LOGIN = "userL";
+    private static final String EMPTY_LOGIN = "";
+    private static final int MIN_AGE = 18;
+    private static final int VALID_AGE = 19;
+    private static final int INVALID_AGE = 17;
+    private static final int ZERO_AGE = 0;
+    private static final int MIN_LENGTH = 6;
+    private static final String ERROR_MESSAGE_INVALID_LOGIN
+            = "Login can't be null or must be more than " + MIN_LENGTH + " characters";
+    private static final String ERROR_MESSAGE_INVALID_PASSWORD
+            = "Password can't be null or must be more than " + MIN_LENGTH + " characters";
+    private static RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        var nullUser = assertThrows(RuntimeException.class,
+                () -> registrationService.register(null));
+        assertEquals("User can`t be a null", nullUser.getMessage());
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        var nullLogin = assertThrows(RuntimeException.class,
```

Comment:
the same here, replace all RuntimeExeption.class

Reply:
This is what happens when you trust a placeholder too much